canonical-websites/www.ubuntu.com
# Flattened and cleaned-up dataprivacy, data-privacy and privacy-policy…

## Done

- Flattened legal data-privacy, dataprivacy and privacy-policy sections
- Updated navigation adding a secondary Data privacy menu
- Updated redirects
- Added some special redirects requested by legal team
- Removed more absolute links

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- look at the files in the /legal directory and compare to this [spreadsheet](https://docs.google.com/spreadsheets/d/12kuuGgwRqN3k7wlRc4QKIuZifjvxbynizO_LEjKjBq4/edit#gid=0)
* test the link [/legal/data-privacy](http://0.0.0.0:8001/legal/data-privacy) and [the old link](http://0.0.0.0:8001/legal/dataprivacy) redirects also test that [/data-privacy](http://0.0.0.0:8001/data-privacy) redirects as well
* test the link [/legal/data-privacy](http://0.0.0.0:8001/legal/data-privacy) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/privacy-policy) redirects also test that [/privacy-policy](http://0.0.0.0:8001/privacy-policy) redirects as well
* test the link [/legal/data-privacy](http://0.0.0.0:8001/legal/data-privacy) and [the old link](http://0.0.0.0:8001/legal/dataprivacy) redirects also test that [/data-privacy](http://0.0.0.0:8001/data-privacy) redirects as well
* test the link [/legal/data-privacy/enquiry](http://0.0.0.0:8001/legal/data-privacy/enquiry) and [the old link](http://0.0.0.0:8001/legal/data-privacy-enquiry) redirects

## Issue / Card

Fixes #4194
